ci-templates/prepare-cache.yaml

```diff
@@ -0,0 +1,4 @@
+steps:
+- script: find $(MAVEN_CACHE_FOLDER)io/quarkus/ -mindepth 1 -maxdepth 1 \( ! -name http -a ! -name gizmo -a ! -name security \) -exec rm -rf {} \;
```

Comment:
Can you comment on this one a little?

Reply:
I don't see what is unclear ;). Yeah right, I will add a comment.